### PR TITLE
IT Index Created

### PR DESCRIPTION
**Pull request by dzpex**

### CODE REVIEW

File: it-index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      Disponibile per Windows, macOS o Linux per connettersi ai server Jamulus di tutto il mondo, gratuitamente tramite una normale connessione internet a banda larga.
+      Oppure creare un server privato. Jamulus è progettato per un suono di alta qualità a bassa latenza, al fine di suonare con altri a tempo. Dai uno sguardo alla pagina  <a href="wiki/Demos" target="_blank"
+        rel="noreferrer"> che spiega come gli utenti usano Jamulus</a>!
+    </div>
+  </div>
+</div>
+<div class="fx-row fx-row-center-xs" id="bannercontainer">
+  <div class="fx-col-100-xs">
+    <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">
+      <img alt="Jamulus Banner. Links to getting started page" src="{{ '/assets/img/jamulusbannersmall.png' | relative_url }}" id="jamulusbanner" loading="lazy" />
```
Comment (ann0see) [inline it-index.html:29]: Thanks. Could you please also translate alt texts?

File: it-index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Jamulus ‒ Suonare Online. Tra Amici. In libertà."
+lang: "it"
+permalink: /
+layout: mainhomepage
+ldjAppCategory: "Music"
```
Comment (ann0see) [inline it-index.html:6]: Looks good. 

Is the Italian word for music the same as in English?

**Review by ann0see — CHANGES_REQUESTED** [review]
For me everything else seems to be ok. @gilgongo any issues?

File: it-index.html (hunk 2 above)
Comment (dzpex) [reply it-index.html]: I applied the correction on the index page. Thanks!

**Review by ann0see — APPROVED** [review]

File: it-index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+permalink: /
+layout: mainhomepage
+ldjAppCategory: "Musica"
+ldjSlogan: "Jamulus ‒ Suonare Online. Tra Amici. In libertà."
+ldjCreator: "Volker Fischer ed altri collaboratori"
+metadescription: "Jamulus è un software opensource che permette ai musicisti di suonare, provare o partecipare ad una jam, in tempo reale tramite Internet."
+mAltProgIcon: "Jamulus icon"
+mTSlogan: "Suonare Online. Tra Amici. In libertà."
+mTGetStartedNow: "Comincia a Suonare!"
+mHTMLDownloadNow: 'Advanced: <a href="https://sourceforge.net/projects/llcon/files/latest/download" target="_blank" id="dld_frm_sf" rel="noreferrer">Scarica Ora</a>'
```
Comment (ann0see) [inline it-index.html:13]: By the way: Is "Advanced" ok here?

File: it-index.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+<div class="fx-row fx-row-center-xs" id="bannercontainer">
+  <div class="fx-col-100-xs">
+    <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">
+      <img alt="Jamulus Banner. Links alla pagina Primi Passi" src="{{ '/assets/img/jamulusbannersmall.png' | relative_url }}" id="jamulusbanner" loading="lazy" />
+    </a>
+  </div>
+</div>
+<div class="fx-row fx-row-center-xs">
+  <div class="fx-col-100-xs fx-col-50-l">
+     <h2>Jamulus worldwide</h2>
```
Comment (ann0see) [inline it-index.html:35]: Same with "Jamulus worldwide"

**Review by ann0see — COMMENTED** [review]
There might still be some changes needed. See my coments

File: it-index.html (hunk 4 above)
Comment (dzpex) [reply it-index.html:35]: I updated Advanced word with right translation but Jamulus Worldwide is untranslatable.

Comment (dzpex) [reply it-index.html:35]: it remain the same in italian

**Comment by dzpex** [comment]
I have ready other wiki page i upload it and i think you can see they on this pull request

**Review by ann0see — APPROVED** [review]

File: it-index.html (hunk 4 above)
Comment (ann0see) [reply it-index.html:35]: In the german version I also didn't use "Jamulus worldwide". I translated it into "Who uses Jamulus?" You don't need a word to word translation. Only the message which comes across needs to be similar.

Comment (dzpex) [reply it-index.html:35]: Yes i do not use world by world translation but in this case the meaning of worldwide is more big than who use jamulus, i think.
I think that rappresenting the jamulus community over the internet. One possible way to translate it is "Jamulus users over the world" ("Gli utenti Jamulus nel mondo").

You think is ok?

Comment (ann0see) [reply it-index.html:35]: Yeah. That could be a good option.

Comment (dzpex) [reply it-index.html:35]: ok i update!

**Comment by ann0see** [comment]
Looks good.

**Review by ann0see — APPROVED** [review]

**Comment by ann0see** [comment]
Merged

**Pull request merged**